website/blog/2023-01-02-aluminium.md

```diff
@@ -0,0 +1,769 @@
+- [Add support for type definition](#add-support-for-type-definition)
+- [Run main classes in terminal when using code lenses](#run-main-classes-in-terminal-when-using-code-lenses)
+- [Allow rename for local names for non compiled code](#allow-rename-for-local-names-for-non-compiled-code)
+- [Enable run and debug Scala CLI scripts in Metals](#enable-run-and-debug-scala-cli-scripts-in-metals)
+- [Refactor package definition when moving files between packages](#refactor-package-definition-when-moving-files-between-packages)
+- [Scala debugger improvements](#scala-debugger-improvements)
+
+## Converting sbt style to mill style dependencies
+
+Thanks to [LaurenceWarne](https://github.com/LaurenceWarne) there is a new code
+action that allows users to automatically convert sbt style defined dependencies
+to Mill ones whenever pasting in scripts or mill files.
+
+For example if we have an incorrect `build.sc` file:
+
+```scala
+object MyModule extends ScalaModule {
+  def ivyDeps = Agg(
+    "org.scalameta" %% "metals" % "1.0"
+  )
+}
+```
+
+Metals will now suggest to convert the incorrect line with the dependency to a
+correct one when hovering or moving the cursor to its location. Afterwards to
+code will turn to:
+
+```scala
+object MyModule extends ScalaModule {
+  def ivyDeps = Agg(
+    ivy"org.scalameta::metals:1.0"
+  )
+}
+```
+
+Similarly, it will also help with ammonite scripts and turn:
+
+```scala
+import $ivy."org.scalameta" %% "metals" % "1.0""
+```
+
+into:
+
+```scala
+import $ivy.`org.scalameta::metals:1.0`
+```
+
+## Additional completions for dependencies
+
+In the previous version of Metals additional library dependency suggestions were
+add in Ammonite scripts and worksheets. This version expands the number of
+places where you can use the feature:
+
+- ScalaCLI scripts and files when writing `//> using lib ...`
+- When writing `ivy"dependency..."` in build.sc file
+- Inside sbt files when adding dependencies.
+
+This feature allows you to explore the available libraries and their versions
+without exiting the comforts of your editor.
+
+Thanks [jkciesluk](https://github.com/jkciesluk) and
+[LaurenceWarne](https://github.com/LaurenceWarne) for all the great work!
+
+## Actionable diagnostics
+
+Actionable diagnostics is a new idea within the Scala tooling ecosystem that
+focuses on providing users with a potential fix for any errors or warnings that
+may show up. This reduces the amount of time users will spend on fixing their
+code and help them be more productive in daily development.
+
+The first experiment within that area was done by
+[lwronski](https://github.com/lwronski) in
+[ScalaCLI](https://scala-cli.virtuslab.org/). When using the newest ScalaCLI you
+will see warning whenever a library has a newer version available and have a
+quick fix suggested via code action.
+
+![actionable](https://i.imgur.com/3D9WcQb.gif) These warnings can be disabled
+within ScalaCLi using:
+
+```
+scala-cli config actions false
+```
```

Comment:
Just to clarify, when someone is using Metals this will turn it off for the workpace? Globally? Only for the active session?

Reply:
Actually, I wanted to confirm it still with @lwronski Is this the correct  command even?

Reply:
This will disable actionable diangostic globally.

Reply:
Let's then add a note here that just says:

```
NOTE: that if you do this, you'll turn off actionable diagnostics from scala-cli globally until this command is ran again with `true`.
```

Or something like that.

Reply:
Good idea, it's added.